packages/console/src/ds-components/ConfirmModal/index.tsx

```diff
@@ -50,7 +55,10 @@ function ConfirmModal({
         title={title}
         footer={
           <>
-            {onCancel && <Button title={cancelButtonText} onClick={onCancel} />}
+            {customActionButton}
+            {!customActionButton && onCancel && (
```

Comment:
This logic is painful. And after offline discussion, I think we should request a design change, to not show a "Contact Us" button on the confirm modal